runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -126,6 +127,28 @@ public PythonEvaluationResult eval(String expression, String prepareEnvironmentS
         }
     }
 
+    public PythonEvaluationResult test(String expression, String prepareEnvironmentScript,
+                                       Map<String, Serializable> context, long timeout) {
+        TempEvalEnvironment tempEvalEnvironment = null;
+        try {
+            String pythonPath = checkPythonPath();
+            tempEvalEnvironment = generateTempResourcesForEval();
+            String payload = generatePayloadForEval(expression, prepareEnvironmentScript, context);
+            addFilePermissions(tempEvalEnvironment.parentFolder);
```

Comment:
This code is probably duplicated with the `eval`'s method one. Can you please extract it in a private method ?

Reply:
done